src/cozmo/conn.py

```diff
@@ -356,7 +356,9 @@ def _handle_ui_device_connected(self, msg):
         self._is_ui_connected = True
         self.dispatch_event(EvtConnected, conn=self)
         self.anim_names.refresh()
-        logger.info("UI device connected")
+        logger.info('App connection established.  sdk_version=%s  '
+                'cozmoclad_version=%s app_build_version=%s',
```

Comment:
i can remove the extra space from the end of the first line, but the 2 spaces afte rthe period was deliberate.

Not too worried about line wrapping; it's a log file, not an interactive prompt.. i'd rather have related data grouped together on a single line than split it across multiple disconnected lines as analyzing it later becomes much more difficult.